cmd/vgmanager/vgmanager.go

```diff
@@ -150,7 +150,6 @@ func run(cmd *cobra.Command, _ []string, opts *Options) error {
 	if err := loadConfFile(ctx, lvmdConfig, lvmd.DefaultFileConfigPath); err != nil {
 		opts.SetupLog.Error(err, "lvmd config could not be loaded, starting without topolvm components and attempting bootstrap")
 	} else {
-		topoLVMD.Containerized(true)
 		lvclnt, vgclnt := topoLVMD.NewEmbeddedServiceClients(ctx, lvmdConfig.DeviceClasses, lvmdConfig.LvcreateOptionClasses)
```

Comment:
Why do we remove this?

Reply:
topolvm has removed this flag because the nsenter call is a noop if not run in a container so they dont need the check anymore